src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -407,32 +407,32 @@ def _download_blob(blob_service, container, destination_folder, normalized_blob_
 
 def storage_blob_upload_batch(cmd, client, source, destination, pattern=None,  # pylint: disable=too-many-locals
                               source_files=None, destination_path=None,
-                              destination_container_name=None, blob_type=None,
+                              container_name=None, blob_type=None,
```

Comment:
My suggestion is to keep the origin `destination_container_name` so that you don't have to change the validator as the validator is also used by other function